include/http/request_handler.js

```diff
@@ -303,7 +303,7 @@ module.exports = function RequestHandlerModule(pb) {
              * @method writeResponse
              * @param {Object} data
              */
-        writeResponse(data) {
+        async writeResponse(data) {
```

Comment:
If this async is here just because you are calling the settings for tn_auth, I think you could simplify this by using the siteObj settings instead (and attaching the new tn_auth setting to the site instead). See line 367 for an en example:

`const prefix = this.req && this.req.siteObj && this.req.siteObj.prefix;`

No real need to make more changes here than required.

I tested these changes but I could not find issues, but I'd be happier if we avoid the async.